Dockerfile

```diff
@@ -6,8 +12,10 @@ RUN apk add --update gcc>=9.3.0 g++>=9.3.0 alpine-sdk
 WORKDIR /go/src/app/
 
 COPY . .
+COPY --from=js /usr/src/app ./controller/app
 # Fetch dependencies.
 RUN go get -d -v ./...
+RUN go generate ./...
```

Comment:
should this not be committed to master as-is? if we're worrying about forgetting to re-run go generate, maybe we should make that a CI check instead.

Reply:
I worded my first sentence weird. I mean that the output of `go generate` should always be in master, so I don't think re-running go generate gets us anywhere.

Speaking of which, I don't get the `go get -d -v ./...` either. Modules will download dependencies as needed in the next step already.

Reply:
I feel like `script.js` as a generated code that changes with every commit to frontend code is easier to build for docker rather than have in each commit as a way to not have to deal with merge conflicts

Reply:
Ah, I see. Maybe move it out of `go generate`, and into a build script of some sort, then? Because it is generally understood that the output of `go generate ./...` should be committed, so this will end up being confusing or problematic sooner or later. You can include its output dir in gitignore, though that adds to the confusion, if you ask me :)

Reply:
the output is in gitignore :)